src/make.js

```diff
@@ -0,0 +1,13 @@
+function make(input) {
```

Comment:
@zm1try What about make(4, 5, 5, 5, 6, sub) ?

Reply:
@manuminsk Had realized and have solved the problem. Thanks. Can I push the fix?